.github/workflows/build_css.yml

```diff
@@ -0,0 +1,73 @@
+name: Compile SCSS
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  build_css:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Sass Build
+        uses: gha-utilities/sass-build@v0.3.4
+        with:
+          source: ~/scss/styles.scss
+          destination: ~/css/styles.min.css
```

Comment:
Ah! Sorry. I didn't spot this when you first asked, because I wasn't looking. In Linux/MacOS, ~ normally means "the root of the user's home directory" (I know it has a strange meaning in sass-land).

The other thing to note is that I don't *think* that github checks out your repo by default. You have to do that explicitly with actions/checkout. Can pair in a bit.